waspc/src/Wasp/Message.hs

```diff
@@ -0,0 +1,5 @@
+module Wasp.Message (Message (..), SendMessage) where
+
```

Comment:
Let's explain the reasoning and what this Message stands for. How it should be used, where, ... .

Maybe not just `Message`? Maybe `WaspMessage`? Or smth else? `CompilerMessage`?